### PR TITLE
#874 add miovision breaks (discontinuities) table

### PR DESCRIPTION
**Pull request by gabrielwol**
## What this pull request accomplishes:
- create `miovision_api.breaks` table. 

## Issue(s) this solves:

- #874 

## What, in particular, needs to reviewed:
- Copied basic structure from ecocounter. 
- I think it would be best to create this table, start adding issues, and continue discussion from there. 
- Should we actually name it `discos`? 

## What needs to be done by a sysadmin after this PR is merged
- [x] create table!
- [x] add some starting entries: https://github.com/CityofToronto/bdit_data-sources/issues/846#issuecomment-1947241738
- [x] @Nate-Wessel evaluate all the null intersection_uid entries and evaluate if they belong in AR or Disco table (or both).

### CODE REVIEW

File: volumes/miovision/sql/README.md [hunk 1]
```diff
@@ -317,6 +318,19 @@ Used to determine the maximum acceptable gap for use in `unacceptable_gaps` tabl
...
+The discontinuity table contains `Moments in time when data collection methods changed in such a way that we would expect clear pre- and post-change paradigms that may not be intercomparable`.
+
+| column_name | Comments | data_type | sample |
+| ------ | ----------- |
+| uid    | simple incrementing primary key | serial | 1 |
+| intersection_uid | Reference to intersections table. A null intersection_uid refers to all intersections. | integer
+| classification_uid | Reference to classifications table. A null classification_uid refers to all classifications. | integer
+| break | Moment the change takes place | timestamp
+| give_or_take | approximate bounds if the precise time is not known | interval
+| notes | required description of what changed - be verbose! | text
```
Comment (Nate-Wessel) [inline volumes/miovision/sql/README.md:331]: I don't think this table is formatted properly. It's not rendering right for me.

Comment (gabrielwol) [reply volumes/miovision/sql/README.md]: Whoops. Fixed now.

**Review by Nate-Wessel — APPROVED** [review]
> * I think it would be best to create this table, start adding issues, and continue discussion from there.

I think that's a good approach. I'd expect the structure to change somewhat, but I think it will take a little bit of experience with this to figure out exactly how. 

> * Should we actually name it `discos`?

I'd be open to calling it `breaks` if we think `discontinuities` is a bit too much of a five-dollar-word. (full disclosure: I actually misspelled it as I was typing it just now and had to go back and fix it...)

File: volumes/miovision/sql/table/create-table-discontinuities.sql [hunk 2]
```diff
@@ -0,0 +1,42 @@
+CREATE TABLE miovision_api.discontinuities (
+    uid serial PRIMARY KEY,
+    intersection_uid integer,
+    classification_uid integer,
```
Comment (Nate-Wessel) [inline volumes/miovision/sql/table/create-table-discontinuities.sql:4]: Should add `REFERENCES` constraints on the appropriate tables.

Comment (gabrielwol) [reply volumes/miovision/sql/table/create-table-discontinuities.sql:4]: Does that support nulls? 🤷‍♂️

Comment (Nate-Wessel) [reply volumes/miovision/sql/table/create-table-discontinuities.sql:4]: It does! (We don't? We should...)

Comment (Nate-Wessel) [reply volumes/miovision/sql/table/create-table-discontinuities.sql:4]: Just double checked and `anomalous_ranges` actually does have these constraints already. 
`miovision_qc_intersection_uid_fkey` and
`miovision_qc_classification_uid_fkey`

Comment (gabrielwol) [reply volumes/miovision/sql/table/create-table-discontinuities.sql:4]: Whoops don't know how I missed those. I will add to this table too.

**Review by chmnata — APPROVED** [review]

**Comment by Nate-Wessel** [comment]
> @Nate-Wessel can you see the one remaining action item before we close the PR?

I'll put that on my to-do list for Monday!

**Comment by Nate-Wessel** [comment]
@gabrielwol I took a look and decided to just breakify the anomalies related to the streetcar reclassification. The rest look like legit anomalies to me. I removed those two from `anomalous_ranges` and added them to `breaks`.

**Pull request merged**